conn.go

```diff
@@ -52,51 +52,82 @@ type Conn struct {
+	InvalidPurpose                              VerifyResult = C.X509_V_ERR_INVALID_PURPOSE
+	CertUntrusted                               VerifyResult = C.X509_V_ERR_CERT_UNTRUSTED
+	CertRejected                                VerifyResult = C.X509_V_ERR_CERT_REJECTED
+	SubjectIssuerMismatch                       VerifyResult = C.X509_V_ERR_SUBJECT_ISSUER_MISMATCH
+	AkidSkidMismatch                            VerifyResult = C.X509_V_ERR_AKID_SKID_MISMATCH
+	AkidIssuerSerialMismatch                    VerifyResult = C.X509_V_ERR_AKID_ISSUER_SERIAL_MISMATCH
+	KeyusageNoCertsign                          VerifyResult = C.X509_V_ERR_KEYUSAGE_NO_CERTSIGN
+	UnableToGetCrlIssuer                        VerifyResult = C.X509_V_ERR_UNABLE_TO_GET_CRL_ISSUER
+	UnhandledCriticalExtension                  VerifyResult = C.X509_V_ERR_UNHANDLED_CRITICAL_EXTENSION
+	KeyusageNoCrlSign                           VerifyResult = C.X509_V_ERR_KEYUSAGE_NO_CRL_SIGN
+	UnhandledCriticalCrlExtension               VerifyResult = C.X509_V_ERR_UNHANDLED_CRITICAL_CRL_EXTENSION
+	InvalidNonCa                                VerifyResult = C.X509_V_ERR_INVALID_NON_CA
+	ProxyPathLengthExceeded                     VerifyResult = C.X509_V_ERR_PROXY_PATH_LENGTH_EXCEEDED
+	KeyusageNoDigitalSignature                  VerifyResult = C.X509_V_ERR_KEYUSAGE_NO_DIGITAL_SIGNATURE
+	ProxyCertificatesNotAllowed                 VerifyResult = C.X509_V_ERR_PROXY_CERTIFICATES_NOT_ALLOWED
+	InvalidExtension                            VerifyResult = C.X509_V_ERR_INVALID_EXTENSION
+	InvalidPolicyExtension                      VerifyResult = C.X509_V_ERR_INVALID_POLICY_EXTENSION
+	NoExplicitPolicy                            VerifyResult = C.X509_V_ERR_NO_EXPLICIT_POLICY
+	UnnestedResource                            VerifyResult = C.X509_V_ERR_UNNESTED_RESOURCE
+	ApplicationVerification                     VerifyResult = C.X509_V_ERR_APPLICATION_VERIFICATION
+	CertificateAuthorityKeyTooSmall             VerifyResult = C.X509_V_ERR_CA_KEY_TOO_SMALL
+	CertificateAuthorityMessageDigestTooWeak    VerifyResult = C.X509_V_ERR_CA_MD_TOO_WEAK
+	RevocationListPathValidationError           VerifyResult = C.X509_V_ERR_CRL_PATH_VALIDATION_ERROR
+	DomanNameAuthenticationNamedEntitiesNoMatch VerifyResult = C.X509_V_ERR_DANE_NO_MATCH
+	RevocationListDifferentScope                VerifyResult = C.X509_V_ERR_DIFFERENT_CRL_SCOPE
+	EndEntityKeyTooSmall                        VerifyResult = C.X509_V_ERR_EE_KEY_TOO_SMALL
+	EmailMismatch                               VerifyResult = C.X509_V_ERR_EMAIL_MISMATCH
+	ExcludedViolation                           VerifyResult = C.X509_V_ERR_EXCLUDED_VIOLATION
+	HostnameMismatch                            VerifyResult = C.X509_V_ERR_HOSTNAME_MISMATCH
+	InvalidCall                                 VerifyResult = C.X509_V_ERR_INVALID_CALL
+	IpAddressMismatch                           VerifyResult = C.X509_V_ERR_IP_ADDRESS_MISMATCH
```

Comment:
I don't understand this commit. What is going on?

Reply:
This one is about defining all the new constants up to the OpenSSL 1.1.1d version. We are using two of the new ones, but the commit adds all of the x509_V_ERR #defines.